update_factorio.py

```diff
@@ -51,25 +51,30 @@ def get_updater_data(user, token):
 def pick_updates(updater_json, factorio_package, from_version, experimental=False):
     latest = [None, None]
     available_updates = {}
+    current_version = from_version
+    updates = []
+
+    # Get latest stable version
     for row in updater_json[factorio_package]:
         if 'from' not in row:
             latest[0] = row['stable']
             continue
-        available_updates[row['from']] = row['to']
 
-        latest[1] = max(latest[1], row['to'], key=version_key)
-
-    updates = []
+    # Get available updates
+    for row in updater_json[factorio_package]:
+        if not experimental and 'from' in row and row['from'] >= current_version and row['to'] <= latest[0]:
```

Comment:
Ok, it's been a fair while since I wrote this originally, but I'm pretty sure comparing versions as strings doesn't work correctly ("0.13.2" compares as > "0.13.10"); this is why the `version_key` function (stolen from elsewhere, IIRC) exists and breaks the string into a list of ints (which does compare as expected).

Otherwise, I expect this does as expected, so fix the version comparison and I'll go ahead and merge it.